app/backend/app.py

```diff
@@ -12,6 +12,7 @@
 from approaches.readdecomposeask import ReadDecomposeAsk
 from approaches.chatreadretrieveread import ChatReadRetrieveReadApproach
 from azure.storage.blob import BlobServiceClient
+from core.chatgptproxy import ChatGptProxy
```

Comment:
```suggestion

from core.chatgptproxy import ChatGptProxy
```

There should be a line between third-party imports and local imports, to make it clear what comes from where. (The isort extension/package will do this for you)